docs/components/dashboards/dashboard.md

```diff
@@ -7,7 +7,7 @@ indicators such as metrics, trends and other insights.
 ## Usage ---
 
 The purpose of a dashboard is helping users to monitor, measure and analyze
-relevant data from different sources in a *at-a-glance* fashion.
+relevant data from different sources in a _at-a-glance_ fashion.
```

Comment:
frankly this kind of re-formatting perfectly valid MD is churn that adds zero value and makes changes harder to follow